packages/salesforcedx-vscode-core/package.json

```diff
@@ -520,6 +520,14 @@
           "command": "sfdx.force.function.create",
           "when": "sfdx:functions_enabled"
         },
+        {
+          "command": "sfdx.force.function.start",
+          "when": "sfdx:functions_enabled && sfdx:project_opened && resource =~ /.*/functions/[^/]+(/[^/]+\\.(ts|js|json|toml))?$/"
```

Comment:
Are we trying to limit this command in the command palette to only be available if the user has a functions file already open ?

Reply:
Yes, that would be the limitation for now. We could expand it to be anywhere, but we'd need to implement a picker since there could be multiple functions.

Reply:
Sounds good to me, why don't we want to do the same for Invoke and Stop ?

Reply:
For invoke, there's already CodeLens where user can invoke the command. If we wish, we could add it back to command palette when user has json files open.
For stop, there can be at most one container running at this moment, so there's no ambiguity.